src/containers/Tools.vue

```diff
@@ -78,45 +66,40 @@ export default {
     column-gap: 5rem;
     row-gap: 3rem;
 
-    @media only screen and (max-width: 1300px) {
+    @media only screen and (max-width: 1250px) {
       display: flex;
       flex-wrap: wrap;
-      justify-content: space-around;
+      justify-content: flex-start;
     }
   }
 
   &-card {
     &-progress {
       grid-column: 1/2;
       grid-row: 1/2;
+      width: 600px;
+      height: min-content;
```

Comment:
do we think the height is uniform behavior for all cards to factor it out?

Reply:
imo it's fine, if it turns out not fine then we can un-factor it later